gcp/vm/vars.tf

```diff
@@ -0,0 +1,67 @@
+variable "env" {
+    description = "The environment it will serve (prod, staging, etc)"
+    type = "string"
+}
+
+variable "project_name" {
+    description = "GCP project name"
+    type = "string"
+}
+
+variable "region" {
+    description = "GCP region"
+    default = "us-central1"
+}
+
+variable "instance_count" {
+    description = "Number of instances to create"
+    default = 1
+}
+
+variable "instance_name" {
+    type = "string"
+}
+
+variable "instance_type" {
+    type = "string"
+    default = "n1-standard-1"
+}
+
+variable "instance_image" {
+    type = "string"
+    default = "centos-7-v20190423"
+}
+
+variable "instance_project" {
+    type = "string"
+    default = "centos-cloud"
+}
+
+variable "network" {
+    type = "string"
+}
+
+variable "subnetworks" {
+    type = "list"
+}
+
+variable "create_data_volumes" {
+  description = "Whether or not to create additional data disks (GCP Compute PD)"

```

Comment:
If you run Terraform from a terminal, without some required variables defined, it will ask you for an input (in the terminal) to be attributed to that variable.

This description is displayed to help with the input, so it's very important to keep them in all variables (even if it seems unnecessary in the first sight). Even with these intuitive variable names

You can copy some from [here](https://github.com/macunha1/terraform-modules/blob/master/aws/vm/vars.tf) if you like

Reply:
Thanks for that. I'll add a description argument to all variables